packages/blocks/src/embed-html-block/embed-html-block.ts

```diff
@@ -108,6 +108,73 @@ export class EmbedHtmlBlockComponent extends EmbedBlockElement<
     }
   }
 
+  private _fullscreenchange() {
+    if (!this._iframe) return;
+
+    const show =
+      document.fullscreenEnabled && document.fullscreenElement === this._iframe;
+
+    this._iframe.contentWindow?.postMessage({ show }, '*');
+  }
+
+  private _tipHtml() {
+    const computed = getComputedStyle(this);
+    const fontFamily = computed.getPropertyValue('--affine-font-sans-family');
+    const fontSize = computed.getPropertyValue('--affine-font-sm');
+    const borderColor = computed.getPropertyValue('--affine-black-10');
+    const backgroundColor = computed.getPropertyValue('--affine-primary-color');
+    const tipColor = computed.getPropertyValue('--affine-white');
+    const color = computed.getPropertyValue('--affine-text-secondary-color');
+
+    return `<style>
+      #affine-tip {
+        display: none;
+        align-items: center;
+        gap: 4px;
+        height: 28px;
+        position: absolute;
+        left: 50%;
+        bottom: 30px;
+        z-index: 10000;
+        transform: translate3d(-50%, 0, 0);
+
+        font-family: ${fontFamily};
+        color: ${color};
+        font-feature-settings:
+          'clig' off,
+          'liga' off;
+        font-size:  ${fontSize};
+        font-style: normal;
+        font-weight: 500;
+        line-height: 22px; /* 157.143% */
+      }
+      #affine-tip > .key {
+        display: inline-flex;
+        padding: 4px 8px;
+        justify-content: center;
+        align-items: center;
+        gap: 4px;
+        border-radius: 8px;
+        color: ${tipColor};
+        border: 1px solid ${borderColor};
+        background: ${backgroundColor};
+      }
+
+      #affine-tip[data-show] {
+        display: flex;
+      }
+      </style>
+      <div id="affine-tip">
+        Press&nbsp;<span class="key">ESC</span>&nbsp;to&nbsp;close
```

Comment:
The ESC prompt should also be a button that exits fullscreen on click

Reply:
Only if `allow-same-origin` is enabled in the sandbox.

Reply:
1. may use message to tell the top frame to exit fullscreen
2. or perhaps draw the tip overlay outside of the iframe so that no message passing is needed